portfolio/src/main/webapp/script.js

```diff
@@ -56,3 +56,4 @@ function getText() {
   });
 }
 
+
```

Comment:
Remove this line (which should remove this file from the PR).

Reply:
Resolving because removed